packages/protocol/scripts/bash/release-on-devchain.sh

```diff
@@ -63,7 +63,8 @@ yarn ts-node scripts/check-backward.ts sem_check --old_contracts $BUILD_DIR/cont
 
 # From make-release.sh
 echo "- Deploy release of current branch"
-yarn truffle exec --network development ./scripts/truffle/make-release.js --build_directory build/ --report report.json --proposal proposal.json --initialize_data example-initialize-data.json
+INITIALIZATION_FILE=`ls -rt releaseData/initializationData/* | tail -n 1`
```

Comment:
Nit: should be able to remove `-r` and use `head` instead of `tail`. Also, prefer long switches in scripts: `--sort=time` over `-t`, `--lines=` over `-n`.